src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/terminal.utils.tsx

```diff
@@ -329,7 +331,31 @@ const manifestEditButtons = ({
     )
 }
 
-export default function terminalStripTypeData(elementData) {
+const downloadFileFolderButton = (elementData: ElementData): JSX.Element => {
```

Comment:
This should be component since should contain state for showing modal and modal would be opened through this.